internal/handlers/pg/msg_delete.go

```diff
@@ -145,11 +149,26 @@ func (h *Handler) MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 			return nil
 		})
 
+		return err
+	}
+
+	unorderedErrMsgs := new(common.WriteErrors)
+
+	for i := 0; i < deletes.Len(); i++ {
```

Comment:
I'd question how optimal this approach is. If `ordered == true`, we don't need this cycle at all, it's enough to just call all the deletes at once (how we did before). We need one-by-one deletion only if `ordered == false`.

Do you think it's possible to refactor it? To me, it seems to be possible, but we might need something similar to how `h.delete` looks like in the Tigris handler. Thoughts @noisersup @w84thesun @AlekSi?

Reply:
Or maybe I'm actually wrong! If we call all the deletes in the same transaction and one of them fails, then in case of our pg handler, none of the docs will be deleted (because of transaction). But this is not what we want, right? So, I think, I need more documentation somewhere to understand this logic.

Reply:
You right it doesn't look clear. I can try to add some comments, should I cover the whole `MsgDelete` function or just `processQuery` part?

Reply:
Let's cover `processQuery` part, yes.